ts/Grid/Core/Options.ts

```diff
@@ -156,13 +156,18 @@ export interface ColumnsSettings {
     /**
      * The distribution of the columns. If `full`, the columns will be
      * distributed so that the first and the last column are at the edges of
-     * the grid. If `fixed`, the columns will have a fixed width in pixels.
+     * the grid. If `fixed`, the columns will have a fixed width in pixels. If
+     * `mixed`, the column widths will be set according to the `width` option
+     * of each column - CSS styles will be ignored then.
+     * 
+     * If `undefined`, the default column distribution will be used, which is
+     * `mixed` if `width` is set for any column, otherwise `full`.
      *
      * Try it: {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/grid-lite/basic/fixed-distribution | Fixed distribution}
-     *
-     * @default 'full'
+     * 
+     * @default undefined
```

Comment:
Type is undefined, when in line 43, you declare that should be `'full' | 'fixed' | 'mixed'` its a bit unclear.